dev/Repeater/InteractionTests/RepeaterTests.cs

```diff
@@ -22,6 +27,64 @@ namespace Windows.UI.Xaml.Tests.MUXControls.InteractionTests
     [TestClass]
     public class RepeaterTests
     {
-        // Placeholder for interaction tests
+        [ClassInitialize]
+        [TestProperty("RunAs", "User")]
+        [TestProperty("Classification", "Integration")]
+        [TestProperty("TestPass:IncludeOnlyOn", "Desktop")]
+        public static void ClassInitialize(TestContext testContext)
+        {
+            TestEnvironment.Initialize(testContext);
+        }
+
+        public void TestCleanup()
+        {
+            TestCleanupHelper.Cleanup();
+        }
+
+        [TestMethod]
+        public void FocusedItemGetsRecycledUponCollectionReset()
+        {
+            Button navButton = new Button(FindElement.ByName("ItemsRepeater Tests"));
+            navButton.InvokeAndWait();
+
+            Wait.ForMilliseconds(1000);
+
+            navButton = new Button(FindElement.ByName("Collection Changes Demo"));
+            navButton.InvokeAndWait();
+
+            Wait.ForMilliseconds(1000);
```

Comment:
We usually don't allow WaitForSeconds in our tests, they tend to be a source of test instability as if the waiting is required, you can't really be sure you've waited long enough.  Is there another way we can determine when it is safe to proceed?

Reply:
I added those lines to be sure, but the test has not failed when I removed it, so I think it is not needed.